packages/salesforcedx-vscode-core/test/vscode-integration/commands/templates/apexGenerateClass.test.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import { ConfigUtil } from '@salesforce/salesforcedx-utils-vscode';
-import { nls as templatesNls } from '@salesforce/templates-bundle/lib/i18n';
+import { nls as templatesNls } from '@salesforce/templates/lib/i18n';
```

Comment:
The usage like this should not work, but vscode-integration is deprecated now so we just ignore it.